raiden/tests/fixtures/variables.py

```diff
@@ -52,6 +53,27 @@ def events_poll_timeout():
 
 
 @pytest.fixture
+def random_marker():
+    """ A random marker used to identify a pytest run.
+
+    Some tests will spawn a private chain, the private chain will be one or
+    more ethereum nodes on a new subprocesss. These nodes may fail to start on
+    concurrent test runs, mostly because of port number conflicts, but even
+    though the test fails to start it's private chain it may run interacting
```

Comment:
`it's private chain it may run` -> `its private chain may run`

Reply:
`its private chain, it may run`